CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ database via the API
     - Added `@cumulus/api-client/granules.updateGranule` to update granule via the API
   - **CUMULUS-2303**
     - Add translatePostgresProviderToApiProvider method to `@cumulus/db/translate/providers`
+  - **CUMULUS-2304**
```

Comment:
Is it also worth noting that we update the various Lamdas to read rules from PostgreSQL (`sqsMessageConsumer`, `messageConsumer`, `manualConsumer`) even for our own tracking purposes?

Reply:
https://github.com/nasa/cumulus/pull/2296/commits/b9e534a2d458773de296dff70df9ec7317136341